js/src/modal.js

```diff
@@ -69,8 +69,6 @@ const SELECTOR_DIALOG = '.modal-dialog'
 const SELECTOR_MODAL_BODY = '.modal-body'
 const SELECTOR_DATA_TOGGLE = '[data-bs-toggle="modal"]'
 const SELECTOR_DATA_DISMISS = '[data-bs-dismiss="modal"]'
-const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed, .sticky-top'
```

Comment:
`.sticky-top` in not in scrollbar.js
```js
const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed'
```

Reply:
Seems duplication, as the stickies are handled already. 

But in second thought it may be duplication, to handle stickies at, as we already change body padding.
Basically is the same code that was in modal. 
And if we see better, the first method was adding padding and the second added negative margin. So the difference of them was like we didn't handle the stickies at all. I want to say, that may we have to remove stickies handle at all.  stickies are relative positioned things, so the `body` padding is enough

Reply:
I 'll revert the change and we can see it on a next pr